server/webhook-ingest/app/main.py

```
@@ -37,28 +39,28 @@ def verify_github_signature(signature, secret, body):
+        alias="X-Hub-Signature",
+        description="GitHub's HMAC hex digest of the payload, used for verifying the webhook's authenticity",
+    ),
     event_type: str = Header(
-        None, 
+        None,
         alias="X-Github-Event",
         description="The type of event that triggered the webhook, such as 'push', 'pull_request', etc.",
     ),
-    body = Body(...),
-):    
+    body=Body(...),
```

Ruff (0.11.9)

Do not perform function call `Body` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)